tools/windows/DatadogAgentInstaller/WixSetup/Datadog/AgentInstaller.cs

```diff
@@ -50,99 +50,119 @@ public AgentInstaller()
-                        name: ProductFullName,
-                        description: ProductDescription,
-                        // SetProjectInfo throws an Exception is Revision is != 0
-                        // we use Revision = 2 for the next gen installer while it's still a prototype
-                        version: new Version(_agentVersion.Version.Major, _agentVersion.Version.Minor,
-                            _agentVersion.Version.Build, 0)
-                    )
-                    .SetControlPanelInfo(
-                        name: ProductFullName,
-                        manufacturer: CompanyFullName,
-                        readme: ProductHelpUrl,
-                        comment: ProductComment,
-                        contact: ProductContact,
-                        helpUrl: new Uri(ProductHelpUrl),
-                        aboutUrl: new Uri(ProductAboutUrl),
-                        productIconFilePath: new FileInfo(ProductIconFilePath)
-                    )
-                    .DisableDowngradeToPreviousVersion()
-                    .SetMinimalUI(
-                        backgroundImage: new FileInfo(InstallerBackgroundImagePath),
-                        bannerImage: new FileInfo(InstallerBannerImagePath),
-                        // $@"{installerSource}\LICENSE" is not RTF and Compiler.AllowNonRtfLicense = true doesn't help.
-                        licenceRtfFile: new FileInfo(ProductLicenceRtfFilePath)
-                    )
-                    .AddDirectories(
-                        CreateProgramFilesFolder(),
-                        new Dir(new Id("APPLICATIONDATADIRECTORY"), @"%CommonAppData%\Datadog",
-                            new DirPermission("[WIX_ACCOUNT_ADMINISTRATORS]", GenericPermission.All),
-                            new DirPermission("[WIX_ACCOUNT_LOCALSYSTEM]", GenericPermission.All),
-                            new DirPermission("[WIX_ACCOUNT_USERS]", GenericPermission.All),
-                            new DirFiles($@"{EtcSource}\*.yaml.example"),
-                            new Dir("checks.d"),
-                            new Dir(new Id("EXAMPLECONFSLOCATION"), "conf.d",
-                                new Files($@"{EtcSource}\extra_package_files\EXAMPLECONFSLOCATION\*")
-                            )
-                        ),
-                        new Dir(@"%ProgramMenu%\Datadog",
-                            new ExeFileShortcut
-                            {
-                                Name = "Datadog Agent Manager",
-                                Target = "[AGENT]ddtray.exe",
-                                Arguments = "&quot;-launch-gui&quot;",
-                                WorkingDirectory = "AGENT",
-                            }
-                        ),
-                        new Dir("logs")
+                new User(new Id("ddagentuser"), "[DDAGENTUSER_NAME]")
+                {
+                    Domain = "[DDAGENTUSER_DOMAIN]",
```

Comment:
question: Does the user have to specify `DDAGENTUSER_DOMAIN` on the command line now?